doc/src/manual/variables.md

```diff
@@ -110,6 +110,19 @@ A space is required between an operator that ends with a subscript/superscript l
 variable name. For example, if `+ᵃ` is an operator, then `+ᵃx` must be written as `+ᵃ x` to distinguish
 it from `+ ᵃx` where `ᵃx` is the variable name.
 
+
+One special variable name is `_`, which can only be assigned values, but cannot be used to assign values to other variables.
```

Comment:
Any all-underscore variables are treated in the same way. So, instead of picking up the specific example, it would be great if the general rule is described.

Reply:
That was a TIL moment! Thanks for the suggestion, It's fixed now